src/main/java/edu/unc/lib/boxc/migration/cdm/services/BoxctronFileService.java

```diff
@@ -47,8 +49,13 @@ public void generateMapping(BoxctronFileMappingOptions options) throws IOExcepti
         assertProjectStateValid();
         ensureMappingState(options);
 
-        // Gather listing of all potential source file paths to match against
+        // Gather list of all potential source file paths to match against
         List<String> candidatePaths = gatherCandidatePaths(getVelocicroptorDataPath(project.getProjectPath()));
+        // Gather list of all source file paths to exclude
+        List<String> exclusionPaths = new ArrayList<>();
+        if (options.getExclusionsCsv() != null && Files.exists(options.getExclusionsCsv())) {
```

Comment:
I'd suggest letting it fail if the file doesn't exist rather than proceeding like no file was provided, so that the user will be aware they provided an invalid path.